src/index.js

```diff
@@ -14,6 +14,8 @@ axios.defaults.baseURL = process.env.REACT_APP_BASEURL
 
 const store = configureStore()
 
+window.store = store
```

Comment:
does this do anything?

Reply:
thats supposed to be taken out, my bad, lets you check the states whenever!